lib/christmas_tree_formatter.rb

```diff
@@ -1,40 +1,40 @@
 # encoding: UTF-8
 
-require "rspec/core"
-require "rspec/core/formatters/base_text_formatter"
+if defined? RSpec
```

Comment:
I'm a little unhappy with checking for RSpec but AFAIK RSpec won't automatically pick up formatters under an RSpec module. I would look kind of ugly to `--require christmas_tree_formatter/rspec --formatter RSpec::ChristmasTreeFormatter`